mobile/src/main/java/org/hawkular/client/android/activity/LoginActivity.java

```diff
@@ -125,12 +125,12 @@ private void setUpBackendCommunication(Persona persona) {
         }
     }
 
-    private void succeed(Persona persona, Environment environment) {
+    private void succeed(Persona persona) {
         // Save Backend preferences
         Preferences.of(this).host().set(host.trim());
         if (!port.isEmpty()) Preferences.of(this).port().set(Integer.valueOf(port));
         Preferences.of(this).personaId().set(persona.getId());
-        Preferences.of(this).environment().set(environment.getId());
+        //Preferences.of(this).environment().set(environment.getId());
```

Comment:
I didn't want to remove the line because we are still not sure if this is the right fix. I wanted @garg-anuj to check the changes and see if it's ok. So basically the thing is that environment fetching is the part of inventory and inventory is no longer being supported. In login we fetched environment first and then proceeded. I have bypassed environment fetching and the login works perfectly. Maybe you can try to uninstall the app and then build.